app/views/invoices/_form.html.slim

```diff
@@ -10,7 +10,7 @@
       input_html: { class: 'datepicker' }
     = f.input :ends_at, as: :string, placeholder: 'End date',
       input_html: { class: 'datepicker' }
-    = f.input :amount
+    = f.input :amount, disabled: @invoice.invoice_items.any? ? true : false
```

Comment:
@yunixon you don't need to use ternary operator if your condition returns Boolean. Just use `= f.input :amount, disabled: @invoice.invoice_items.any?`